### PR TITLE
Jan Merz Exercise 03

### PR DESCRIPTION
**Pull request by TrueEsel**
Return value of task 3: 1306

### CODE REVIEW

File: Exercises/JanMerz/exercises_03.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            for rank in self.ranks[fromRank:]:
+                self.cards.append(Card(rank, suit))
+
+    def __str__(self):
+        return f", ".join([str(card) for card in self.cards])
+    
+    def __iter__(self):
+        return (card for card in (self.cards))
+
+    def __get__(self, index):
```
Comment (LeMixer) [inline Exercises/JanMerz/exercises_03.py:36]: This does not implement indexing. However the task states you should be able to index the card deck to receive a card in the following manner: `card_at_index = deck[index]`
Try using __getitem__ instead.

File: Exercises/JanMerz/exercises_03.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    def __init__(self, rank, suit):
+        self.rank = rank
+        self.suit = suit
+    
+    def __str__(self):
+        return f"{self.rank} of {self.suit}"
+    
+class FrenchDeck:
+    suits = ["Diamonds", "Hearts", "Spadess", "Clubs"]
```
Comment (LeMixer) [inline Exercises/JanMerz/exercises_03.py:21]: Spades

File: Exercises/JanMerz/exercises_03.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+class SkatDeck(FrenchDeck):
+    def __init__(self):
+        super().__init__(5)
+
+
+# Write some code to test the functionality of both kinds of decks. (You can use `assert` to make sure your classes behave the way you expect them to.)
+french_deck = FrenchDeck()
+
+print(french_deck)
```
Comment (LeMixer) [inline Exercises/JanMerz/exercises_03.py:53]: Not sure why you are printing the deck twice. The second print statement shows that the deck is in fact iterable, but the first print just clutters the console.

**Review by LeMixer — CHANGES_REQUESTED** [review]
Looks good in general. Please revisit the lines I've commented:

File: Exercises/JanMerz/exercises_03.py (hunk 2 above)
Comment (TrueEsel) [reply Exercises/JanMerz/exercises_03.py]: Fixed the Typo

File: Exercises/JanMerz/exercises_03.py (hunk 1 above)
Comment (TrueEsel) [reply Exercises/JanMerz/exercises_03.py]: implemented indexing. Thanks for the tip :)

**Review by LeMixer — APPROVED** [review]
Looks Good

**Pull request merged**